CHANGELOG.md

```diff
@@ -1,3 +1,84 @@
+# TiDB Operator v1.0.1 Release Notes
+
+## v1.0.1 What's New
+
+### Action Required
+
+- ACTION REQUIRED: We fixed a serious bug ([#878](https://github.com/pingcap/tidb-operator/pull/878)) that could cause all `PD` and `TiKV` pods to be accidentally deleted when `kube-apiserver` fails. We strongly suggest users upgrade to `v1.0.1`.
+- ACTION REQUIRED: There is a bug ([#29](https://github.com/pingcap/mydumper/pull/29)) in `mydumper` which may cause the value of a column not to be included in the exported data . We strongly suggest users upgrade the backup image to `pingcap/tidb-cloud-backup:20190828`.
```

Comment:
```suggestion
- ACTION REQUIRED: The backup tool image [pingcap/tidb-cloud-backup](https://hub.docker.com/r/pingcap/tidb-cloud-backup) uses a forked version of [`Mydumper`](https://github.com/pingcap/mydumper). The current version `pingcap/tidb-cloud-backup:20190610` contains a serious bug that could cause the exported data lack of a column. This is fixed in [#29](https://github.com/pingcap/mydumper/pull/29). And the default image used now contains this fixed version. So users using the old version image for backup **must** upgrade to using `pingcap/tidb-cloud-backup:201908028` and do a new full backup to avoid potential data inconsistency.
```